eos/parameters/hadronic-matrix-elements.yaml

```diff
@@ -3366,6 +3366,78 @@ groups:
           central: +0.0
           min:     +0.0
           max:     +0.0
+      'B_s->phiccbar::Re{alpha_0^perp}@GvDV2020' :
+          central: +0.0
+          min:     +0.0
+          max:     +0.0
```

Comment:
Can you add a simple latex representation for these? Skip the GvDV2020, all we need is the B_s->phi part, the polarization, and the order.

Reply:
Yes, I was waiting for issue 171 to be closed before, to avoid painful rebasing.